modules/i18n/src/translations/en-US/portals/console.ts

```diff
@@ -4596,10 +4596,18 @@ export const console: ConsoleNS = {
                             heading: "Group",
                             hint: "Specifies the attribute that identifies the groups at the Connection.",
                             label: "Group Attribute",
-                            message: "Please note that <1>{{ attribute }}</1> attribute will be considered as the default " +
+                            mappedRolesAbsentMessage: "With your current configuration, <1>Group Attribute</1> is not configured. " +
+                                "You can select an attribute from the dropdown.",
+                            mappedRolesPresentMessage: "Please note that <1>{{ mappedRolesClaim }}</1> " +
+                                "will be considered as the default <1> Group Attribute</1> with the current configuration. " +
+                                "You can select an attribute from the dropdown.",
+                            message: "Please note that <1>{{ attribute }}</1> {{ suffix }} will be considered as the default " +
                                 "<1>Group Attribute</1> as you have not added a custom attribute " +
                                 "mapping.",
                             placeHolder: "Select the attribute",
+                            roleMappingDisabledMessage: "<1>Custom Attribute Mapping</1> is disabled in " +
```

Comment:
Update this accordingly

Reply:
Fixed with af68c2afdd1657f85b07719938222114b70cc090